### ISSUE TITLE
Urgent PDM Pricing ChangeSuggestion

### ISSUE THREAD

**Issue by ThisIsNotDex** [InZidiuZ/op-framework-issue-tracker#2477]
**Assurance:**
Yes

**Summary:**
The current state of the vehicle economy, as I'm sure many would agree is unbalanced. In regards to vehicle pricing mainly. One obvious thing to point out, people will buy EDM cars because they are real life cars. Who doesn't want that, but there are plenty of PDM alternatives that can be just as good. The problem is that people might be stuck between two cars, but not choose PDM because of the pricing. Currently, for PDM cars, when people purchase a car the commission to the salesperson is added on, not taken from the vehicles price. This is completely opposite of what happens in real life.

My proposal officially here in GitHub, is to have commission come out of the vehicles cost, not added after the fact. So if someone wanted to purchase/finance a $64,000 car. They would actually be paying $73,600, with the normal 15% commission. These numbers obviously increase with the vehicle price, and make PDM cars unattractive to buyers when they can get an EDM that would be better, and cost less, simply because of commission.

Now I know when a car is sold to PDM, a certain dollar amount is added to the PDM Bank account. I don't know the math on that. But if this pricing were to change, then if the profit that PDM gets could be adjusted to reflect the change. And wouldn't cause a loss in profits, since most likely more cars would be sold, with cars being at 'MSRP'.

**Reason:**
Creates a better balance of vehicles being driven around, and allows players to purchase a wider variety of cars, that are within their budget.

**Comment by CruzClementina** [comment]
TLDR: 
The commission should come out of the price of the vehicle not added after the listed price. 
It's confusing for customers and ends up deterring customers more often than not, due to things being more expensive than they originally thought (even if the commission is dropped).

**Comment by Masik52** [comment]
You could just make the commission 10% (which is around the average %) and I think all parties would be satisfied.  The biggest issue like what was described is you end up spending a substantial amount of money for realistically no reason compared to not having that extra charge at EDM or the tuner shop.

**Comment by twooot** [comment]
Commission is now a fixed 10%. By default, the commission will not be collected. The dealer can set themselves to receive the commission by holding E (this resets after 60 seconds).

**Comment by Ninja2op** [comment]
I brought a motel for 1750 bucks but I can’t enter in pls help

**Comment by King2704** [comment]
> I brought a motel for 1750 bucks but I can’t enter in pls help

this is the wrong place
